source/portable/NetworkInterface/libslirp/MBuffNetworkInterface.c

```diff
@@ -77,76 +78,79 @@ extern void vMBuffNetifBackendDeInit( void * pvBackendContext );
 
 static void vNetifReceiveTask( void * pvParameters );
 
-MBuffNetDriverContext_t xDriverCtx = { 0 };
-
 /**
  * @brief Initialize the MessageBuffer backed network interface.
  *
  * @return BaseType_t pdTRUE on success
  */
-BaseType_t xNetworkInterfaceInitialise( void )
+BaseType_t xNetworkInterfaceInitialise( NetworkInterface_t * pxNetif )
 {
     BaseType_t xResult = pdTRUE;
 
-    if( xDriverCtx.xInterfaceState == pdFALSE )
+    pxNetif->pvArgument = pvPortMalloc( sizeof( MBuffNetDriverContext_t ) );
```

Comment:
@AniruddhaKanhere How are we meant to statically allocate the driver context with this multi-interface setup?